### PR TITLE
Store chat size

### PR DESCRIPTION
**Pull request by vulppine**
## About the PR

Adds the ability for a user set chat size to be stored into the configuration file, so when the game is started up again, the last set chat size is restored. This is per screen type (i.e., default and separated style screen layouts have their own separate stored chat sizes)

**Media**

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Chat size storage. Now you don't have to resize the chat constantly every time you start up the game!

### CODE REVIEW

**Comment by Emisse** [comment]
thank you my child old chat usable finally

**Comment by DrSmugleaf** [comment]
Make the default not a 50/50 split I beg you

**Comment by vulppine** [comment]
> Make the default not a 50/50 split I beg you

ok

image

**Review by DrSmugleaf — CHANGES_REQUESTED** [review]

File: Content.Shared/CCVar/CCVars.cs [hunk 1]
```diff
@@ -1199,6 +1199,11 @@ public sealed class CCVars : CVars
         public static readonly CVarDef<string> UILayout =
             CVarDef.Create("ui.layout", "Default", CVar.CLIENTONLY | CVar.ARCHIVE);
 
+        public static readonly CVarDef<string> DefaultScreenChatSize =
+            CVarDef.Create("ui.default_chat_size", "", CVar.CLIENTONLY | CVar.ARCHIVE);
+
+        public static readonly CVarDef<string> SeparatedScreenChatSize =
+            CVarDef.Create("ui.separated_chat_size", "0.6,0", CVar.CLIENTONLY | CVar.ARCHIVE);
```
[the remark this answers is not in the document]
Comment (vulppine) [reply Content.Shared/CCVar/CCVars.cs:1206]: this should be a good enough size for a 720p screen (which seems to be the default size for SS14 unmaximized)

**Review by DrSmugleaf — APPROVED** [review]

**Pull request merged**